neqo-client/src/main.rs

```diff
@@ -271,26 +247,16 @@ impl Args {
 
 #[derive(Debug, Clone, Copy, PartialEq, Eq)]
 struct VersionArg(Version);
-impl FromStr for VersionArg {
-    type Err = ClientError;
-
-    fn from_str(s: &str) -> Res<Self> {
-        let v = u32::from_str_radix(s, 16)
-            .map_err(|_| ClientError::ArgumentError("versions need to be specified in hex"))?;
-        Ok(Self(Version::try_from(v).map_err(|_| {
-            ClientError::ArgumentError("unknown version")
-        })?))
-    }
-}
 
-#[derive(Debug, StructOpt)]
+#[derive(Debug, Parser)]
 struct QuicParameters {
-    #[structopt(
-        short = "V",
+    #[arg(
+        short = 'v',
```

Comment:
Can we keep the uppercase here?  I like to reserve `-v` to mean "verbose".

Reply:
`-V` is the clap default for `--version`.

Reply:
Yeah, I don't think we should override `--version` either.

`-Q` ? As in `--quic-version`.